en/newsletter_form.ftl

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+newsletter-form-please-enter-a-valid = Please enter a valid email address
+newsletter-form-you-must-agree-to = You must agree to the privacy notice
+newsletter-form-your-email-address = Your email address
+
+# Only localize "yourname". Do not change "@example.com".
+newsletter-form-yournameexamplecom = yourname@example.com
+
+newsletter-form-select-country-or-region = Select country or region
+newsletter-form-your-email-here = YOUR EMAIL HERE
+newsletter-form-format = Format
+newsletter-form-html = HTML
+newsletter-form-text = Text
+newsletter-form-get-firefox-news = Get { -brand-name-firefox } news
+
+# Variables:
+#   $url (url) - link to https://www.mozilla.org/privacy/websites/
+newsletter-form-im-okay-with-mozilla = I’m okay with { -brand-name-mozilla } handling my info as explained in <a href="{ $url }">this Privacy Notice</a>
+
+newsletter-form-we-will-only-send = We will only send you { -brand-name-mozilla }-related information.
+newsletter-form-if-you-havent-previously = If you haven’t previously confirmed a subscription to a { -brand-name-mozilla }-related newsletter you may have to do so. Please check your inbox or your spam filter for an email from us.
```

Comment:
The sentence needs to be reworded: `If.... ` should be a two-part sentence, and the sentence misses a word or two.